src/test/base-route/base-route.test.js

```diff
@@ -6,15 +6,15 @@ import { status } from '../../utils';
 chai.use(chaiHttp);
 chai.should();
 
-const entryRoute = '/';
+const entryRoute = '/api/v1';
```

Comment:
@eokwukwe said asked that base route not use `/api/v1`

Reply:
ok @uthdev, i will return it to `src/index.js`